### PR TITLE
KTLN-250: kotlin spring boot configuration properties

### PR DESCRIPTION
**Pull request by nrsuresh**
https://drafts.baeldung.com/wp-admin/post.php?post=141731&action=edit

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -270,6 +270,7 @@
         <module>kotlin-testing</module>
         <module>machine-learning</module>
         <module>spring-boot-kotlin</module>
+        <module>spring-boot-kotlin-v2</module>
```
Comment (JavierPeris) [inline pom.xml:273]: The module should be named spring-boot-kotlin-2

File: spring-boot-kotlin-v2/pom.xml [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+            <artifactId>jackson-module-kotlin</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-reflect</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-stdlib-jdk8</artifactId>
+        </dependency>
```
Comment (JavierPeris) [inline spring-boot-kotlin-v2/pom.xml:49]: Please remove duplicated dependencies.

File: spring-boot-kotlin-2/pom.xml [hunk 3]
```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>spring-boot-kotlin-v2</artifactId>
+    <name>spring-boot-kotlin-v2</name>
```
Comment (KevinGilmore) [inline spring-boot-kotlin-2/pom.xml:6]: Need to change both artifactId and name to "spring-boot-kotlin-2" (without the "v")

**Comment by JavierPeris** [comment]
retest this please

**Comment by JavierPeris** [comment]
retest this please

**Review by JavierPeris — APPROVED** [review]

**Pull request merged**